doc/nrf/known_issues.rst

```diff
@@ -132,6 +132,15 @@ NCSDK-14235: Timestamps that are sent in cloud messages drift over time
 CIA-604: ATv2 cannot be built for the ``thingy91_nrf9160_ns`` build target with ``SECURE_BOOT`` enabled
   Due to the use of static partitions with the Thingy:91, there is insufficient room in the flash memory to enable both the primary and secondary bootloaders.
 
+.. rst-class:: v2-0-0
+
+CIA-661: ATv2 configured for LwM2M cannot be built for the ``nrf9160dk_nrf9160_ns`` build target with ``CONFIG_NRF_MODEM_LIB_TRACE_ENABLED`` enabled due to memory constraints.
+
+  **Workaround:** Use one of the following workarounds:
+
+  * Use :ref:`secure_partition_manager` instead of TF-M by setting :kconfig:option:`CONFIG_SPM` and disabling :kconfig:option:`CONFIG_BUILD_WITH_TFM`.
```

Comment:
```suggestion
  * Use :ref:`secure_partition_manager` instead of TF-M by setting :kconfig:option:`CONFIG_SPM` to ``y`` and :kconfig:option:`CONFIG_BUILD_WITH_TFM` to ``n``.
```